### PR TITLE
MLOps Doc Updates

### PR DESCRIPTION
**Pull request by iazam6**
Updated/Rewrote BYO doc with images. Added new doc under BYO called Getting Started. This initiative is led by Moran and myself (Ismail). We are working with Sammie to make changes. 

Original google document with edits is here: https://docs.google.com/document/d/1QjSfxOr1Xv1fs3t3tyV8fob9230d_i7cqQT1CDzr9YI/edit#heading=h.r1s4jhyankjm

BYO github code google doc is here:
https://docs.google.com/document/d/1SbVve00UywTjXTT1ra4_jLUNZIr4myosICFDwHGRQQ0/edit#

Getting started github code google doc is here:
https://docs.google.com/document/d/1TjKNxmbIe5LKmaRooSJG4xCe1MJi5Wg5UXLvXskvMSg/edit#

Images in document can be found in google drive folder in case they're needed for edits/errors: https://drive.google.com/drive/folders/1r0Rl5J6wGnQbYlsuPKp04GBfCQR7fZv2?usp=sharing

### CODE REVIEW

File: src/nav/mlops.yml [hunk 1]
```diff
@@ -4,7 +4,11 @@ pages:
   - title: Introduction to model performance monitoring
     path: /docs/mlops/get-started/intro-mlops
   - title: Bring your own data
-    path: /docs/mlops/bring-your-own/mlops-byo
+    pages:
+      - title: Overview
```
Comment (shwartzm-NR) [inline src/nav/mlops.yml:8]: Why do we need this "overview" section?
The mlops-byo content can be attached to the "title: Bring your own data"

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx [hunk 2]
```diff
@@ -5,33 +5,66 @@ redirects:
...
+New Relic's MLOps solution provides a Python library that makes it easy to monitor your machine learning models. New Relic Bring Your Own (BYO) **Machine Learning Model Monitoring** is an offering of the New Relic product suite that allows users to monitor the performance of their own machine learning models, regardless of the platform or framework they are using. This can help users identify issues with their models and take corrective action before those issues impact the accuracy or performance of their applications.
 
+In an increasingly complex digital space, data teams rely heavily on prediction engines to make decisions. New Relic Machine Learning Model Monitoring allows your team to take a step back and look at the whole picture.Our model performance monitoring allows your team to examine your ML models to identify issues efficiently and make decisions. By leveraging NR core features like setting alerts on any metrics and get notified once your ML performance has changed
```
Comment (shwartzm-NR) [inline src/content/docs/mlops/bring-your-own/mlops-byo.mdx:10]: 1. Missing space "the whole picture.Our model
2. "and make decisions. By leveraging" - I think it should be "decisions, by leveraging"

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx [hunk 3]
```diff
@@ -5,33 +5,66 @@ redirects:
...
+
+## Utilize our ML Framework Agnostic [#utilize-ml-framework]
```
Comment (shwartzm-NR) [inline src/content/docs/mlops/bring-your-own/mlops-byo.mdx:12]: I am not sure what this title means

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx [hunk 4]
```diff
@@ -5,33 +5,66 @@ redirects:
...
 
-2. **Data metrics:** Instead of sending all your raw inference data, select the aggregated statistics option over the features, and predictions (for example, `min`, `max`, `average`, or `percentile`). They will be sent automatically as **[metrics](/docs/data-apis/ingest-apis/metric-api/introduction-metric-api/)**.
+<img
+  title="An image of the model count, missing values, and standard deviation in New Relic"
+  alt="An image of the New Relic MLOps dashboard."
+  src={mlops5}
+/>
 
-3. **Custom metrics:** Calculate your own metrics, monitor your model performance or model data, and stream them to New Relic using the `record_metrics` function. They are sent as **[metrics](/docs/data-apis/ingest-apis/metric-api/introduction-metric-api/)**.
+## Want these Benefits? [#get-benefits]
```
Comment (shwartzm-NR) [inline src/content/docs/mlops/bring-your-own/mlops-byo.mdx:62]: Lets change to "How to start monitoring your ML model performance?"

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+ml_performence_monitor_model.record_inference_data(X, y)
+```
+
+### 6. Monitor and alert
+Done! Check your application in the New Relic UI to see the real time data.
+ 
+
+## Examples [#examples]
+We created these notebooks in Google colab so you can try them out:
+  1. Try out an XGBoost model on Boston housing prices dataset. Open in colab.
```
Comment (shwartzm-NR) [inline src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:88]: Missing links (look at the [docs](https://docs.google.com/document/d/1QjSfxOr1Xv1fs3t3tyV8fob9230d_i7cqQT1CDzr9YI/edit#)

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+  2. Try out how to simulate 24 hours of model inference data using New Relic MLOps. Open in colab
```
Comment (shwartzm-NR) [inline src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:89]: Missing links

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+y = my_model.predict(X)
+```
+
+### 5. Record
+```python
+ml_performence_monitor_model.record_inference_data(X, y)
+```
+
+### 6. Monitor and alert
+Done! Check your application in the New Relic UI to see the real time data.
```
Comment (shwartzm-NR) [inline src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:83]: Missing link

File: src/nav/mlops.yml (hunk 1 above)
Comment (homelessbirds) [reply src/nav/mlops.yml]: If you want an overarching tab that says, "Bring your own data" then the mlops-byo content will need to have a different title. I am open to other names!

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/mlops/bring-your-own/mlops-byo.mdx:10]: Done

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/mlops/bring-your-own/mlops-byo.mdx:62]: done

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx (hunk 3 above)
Comment (iazam6) [reply src/content/docs/mlops/bring-your-own/mlops-byo.mdx]: What would you like the title to be replaced with? This was the original title in the doc you created

File: src/nav/mlops.yml (hunk 1 above)
Comment (homelessbirds) [reply src/nav/mlops.yml]: This was resolved in a google doc

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+
+## Implementation [#implementation]
+
+This guide will take you step by step for everything needed to start monitoring your Machine Learning models
+
+### 1. Set Your Environment Variable
+
+[Get your License key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) (also referenced as `ingest - license`) and set it as environment variable: `NEW_RELIC_INSERT_KEY`. [Click here](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key) for more details and instructions.
+Are you reporting data to the New Relic EU region? click [here](https://github.com/newrelic-experimental/ml-performance-monitoring#eu-account-users) for more instructions.
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:42]: done

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:89]: done

File: src/content/docs/mlops/bring-your-own/mlops-byo.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/mlops/bring-your-own/mlops-byo.mdx]: This has been done.

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx (hunk 5 above)
Comment (homelessbirds) [reply src/content/docs/mlops/bring-your-own/getting-started-byo.mdx]: done

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**